src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -141,61 +151,77 @@ export class SearchMenu extends React.Component {
-          acceptCharset="UTF-8"
-          action={this.state.searchAction}
-          id="search"
-          method="get"
-          onSubmit={this.logSearchEvent(undefined)}
-        >
-          <label htmlFor="query" className="usa-sr-only">
-            Search:
-          </label>
-
-          <div className="va-flex">
-            <input
-              autoComplete="off"
-              ref="searchField"
-              className="usagov-search-autocomplete vads-u-margin-right--0p5"
-              id="query"
-              name="query"
-              type="text"
-              onChange={this.handleInputChange}
-              value={this.state.userInput}
-            />
-
-            <button
-              type="submit"
-              disabled={!validUserInput}
-              className="vads-u-margin-left--0p25"
-            >
-              <IconSearch color="#fff" />
-              <span className="usa-sr-only">Search</span>
-            </button>
+      <Downshift
+        id="search"
+        inputValue={this.state.userInput}
+        onSelect={item => this.handleSearchEvent(item)}
+        itemToString={item => {
+          return item;
+        }}
+        isOpen={this.props.isOpen}
+      >
+        {({
+          getInputProps,
+          getItemProps,
+          isOpen,
+          highlightedIndex,
+          selectedItem,
+        }) => (
+          <div className="typeahead-search vads-u-padding--0 vads-u-padding-x--0p5 medium-screen:vads-u-padding--0">
+            <div className="va-flex vads-u-align-items--center vads-u-justify-content--center vads-u-padding-x--0p5 medium-screen:vads-u-padding--0">
+              <label htmlFor="query" className="usa-sr-only">
+                Search:
+              </label>
+              <input
+                autoComplete="off"
+                className="usagov-search-autocomplete  vads-u-flex--4 vads-u-margin-left--1 vads-u-margin-right--0p5 vads-u-margin-y--1  vads-u-width--full"
+                name="query"
+                aria-controls="suggestions-list"
+                {...getInputProps({
+                  type: 'text',
+                  onChange: this.handleInputChange,
+                  'aria-labelledby': 'site search',
+                  id: 'query',
+                })}
+              />
+              <button
+                type="submit"
+                disabled={!validUserInput}
+                className="vads-u-margin-left--0p5 vads-u-margin-y--1 vads-u-margin-right--1 vads-u-flex--1"
+                onClick={() => this.handleSearchEvent()}
+              >
+                <IconSearch color="#fff" />
+                <span className="usa-sr-only">Search</span>
+              </button>
+            </div>
+            {isOpen && (
+              <div
+                id="suggestions-list"
+                className="vads-u-margin-top--0p5"
+                role="listbox"
+              >
+                {this.state.suggestions.map((suggestion, index) => (
```

Comment:
If suggestions is `undefined` or `null` for whatever reason, this will throw a runtime error. You could update this to `this.state.suggestions?.map` if you wanted! 🙂

Reply:
There is a default value for suggestions when it is initialized, but this is a good idea. Will be added in the next push.

Reply:
I used this implementation as it was what other typeahead options were using in other areas, I will check with accessibility to see if <ul> / <li> will work here. @jenstrickland and @joshkimux do you have any thoughts?

Reply:
Hmmm, I thought that [`<datalist>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) was better for accessibility

Reply:
```<datalist>``` doesn't work for our event logging requirements. After accessibility did a review we came to the conclusion that the Downshift component best fit our use case at this time.

Reply:
Apologies for the delay in response. It would be preferable to use semantic elements instead of `<div>`. I'd like to test it in a screen reader to see how it sounds the way it's built. With the styling, the CSS will wipe out the list semantics of a `<ul> + <li>` pattern. The `role="listbox"` that you've used should work(https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role). Thank you! Just want to validate in the browser+screenreader pairings we have to support. When this is in a review environment, would you tag me to review in JAWS, NVDA, and VoiceOver, please? :)